src/Converter/Feature2Mesh.js

```diff
@@ -321,31 +316,33 @@ function featureToExtrudedPolygon(feature, options) {
     const indices = [];
     const totalVertices = ptsIn.length / 3;
 
-    vertices.minAltitude = Infinity;
-
     const batchIds = options.batchId ?  new Uint32Array(vertices.length / 3) : undefined;
     let featureId = 0;
 
+    const z = options.GlobalZTrans - feature.altitude.min;
+    const globals = { fill: true };
+    let fillColor;
+
     for (const geometry of feature.geometries) {
-        const altitude = getProperty('altitude', options, 0, geometry.properties);
-        const extrude = getProperty('extrude', options, 0, geometry.properties);
+        const context = { globals, properties: () => geometry.properties };
+        const style = feature.style.drawingStylefromContext(context);
 
-        const colorTop = getProperty('color', options, randomColor, geometry.properties);
-        color.copy(colorTop);
-        color.multiplyScalar(0.6);
+        fillColor = toColor(style.fill.color);
+        color.copy(fillColor);
+        color.multiplyScalar(0.5);
```

Comment:
Maybe you could add a comment to specify that this `color` is applied to the bottom surface of extruded polygons, and that `fillColor` is applied to the top surface.

Reply:
added